docs/user_guide/trouble_shooting.md

```diff
@@ -144,6 +144,43 @@ kubectl scale deployment yunikorn-scheduler -n yunikorn --replicas=0
 kubectl scale deployment yunikorn-scheduler -n yunikorn --replicas=1
 ```
 
+## Gang Scheduling
+
+### 1. No placeholders created, app's pods are pending
+
+*Reason*: This is usually because the taskGroups definition is invalid, and the app is rejected by the scheduler.
```

Comment:
It can be because the total resource request exceeds the queue resources. In this case the app is also rejected

Reply:
Good point, added to the doc